devices/enocean.js

```diff
@@ -0,0 +1,48 @@
+const exposes = require('../lib/exposes');
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const e = exposes.presets;
+
+module.exports = [
+    {
+        fingerprint: [{modelID: 'GreenPower_2', greenPowerKey: '62149.46.197.242'},
+            {modelID: 'GreenPower_2', greenPowerKey: '0.46.197.242'}],
+        model: 'PTM 215Z',
+        vendor: 'EnOcean',
+        description: 'Pushbutton transmitter module',
+        fromZigbee: [fz.enocean_ptm215z],
+        toZigbee: [],
+        exposes: [e.action(['press_1', 'release_1', 'press_2', 'release_2', 'press_3', 'release_3', 'press_4', 'release_4',
+            'press_1_and_3', 'release_1_and_3', 'press_2_and_4', 'release_2_and_4'])],
+        whiteLabel: [
+            {vendor: 'Niko', description: 'Dimmer switch for Hue system', model: '91004'},
```

Comment:
@fredericseiler made some updates, added the `PTM 216Z`. Suprisingly my Niko 91004 had a different `greenPowerKey` (`0.46.197.242`). Did you test this with different devices with the same model?